presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -861,7 +862,7 @@ public void testDecimalBackedByINT32()
     public void testDecimalBackedByINT64()
             throws Exception
     {
-        for (int precision = 1; precision <= MAX_PRECISION_INT64; precision++) {
+        for (int precision = 4; precision <= MAX_PRECISION_INT64; precision++) {
```

Comment:
why change here?

Reply:
Decimal(3, 0) cannot keep -1000..1000 ints

Reply:
ok, make it separate commit